src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1139,10 +1140,10 @@ public static bool IsPossibleTupleElementNameContext(this SyntaxToken token, int
 
             if (token.IsKind(SyntaxKind.OpenParenToken))
             {
-                return token.Parent.IsKind(SyntaxKind.ParenthesizedExpression) || token.Parent.IsKind(SyntaxKind.TupleExpression);
+                return token.Parent.IsKind(SyntaxKind.ParenthesizedExpression) || token.Parent.IsKind(SyntaxKind.TupleExpression, SyntaxKind.TupleType);
```

Comment:
It's possible this method is misnamed. It has already been used for more than just tuple element name context.
Maybe rename to `IsPossibleTupleContext`?